gui/widgets/edittext.cpp

```diff
@@ -132,14 +131,20 @@ void EditTextWidget::startEditMode() {
 
 void EditTextWidget::endEditMode() {
 	releaseFocus();
-
+	
+	sendCommand(_ExitTxtCmd, 0);
 	sendCommand(_finishCmd, 0);
 }
 
 void EditTextWidget::abortEditMode() {
 	setEditString(_backupString);
 	sendCommand(_cmd, 0);
+	
 	releaseFocus();
 }
 
+Common::String EditTextWidget::getEditString(){
```

Comment:
Missing space after brackets

Reply:
The commit messages themselves don't conform to our guidelines.. you forgot to include the 'GUI: ' prefix in this case, since they're affecting the GUI code. Please rebase your commits to add in the prefix.